packages/node_modules/@webex/plugin-memberships/src/memberships.js

```diff
@@ -70,6 +70,31 @@ const Memberships = WebexPlugin.extend({
 
     /* eslint-disable no-case-declarations */
     switch (activity.verb) {
+      case SDK_EVENT.INTERNAL.ACTIVITY_VERB.CREATE:
+        const membershipCreatedEventDataArray =
+          activity.object.participants.items.map(
+            (participant) => {
```

Comment:
I see you're a fan of prettier (I am too 😄).
But for now, you can do on the same line until we put prettier in the repo
So this
`activity.object.participants.items.map((participant) => {`
is fine for now for readability

Reply:
Latest commit addresses this style concern.